worlds/timespinner/Regions.py

```diff
@@ -45,6 +45,10 @@ def create_regions(world: MultiWorld, player: int, locations: Tuple[LocationData
         create_region(world, player, locations_per_region, location_cache, 'Space time continuum')
     ]
 
+    throwIfAnyLocationIsNotAssignedToARegion(regions, locations_per_region.keys())
```

Comment:
this might be a use case for "if __debug__:"
some more info:
https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement

Reply:
Hmmz why would you suggest this to de a debug contraint? this should never happen unless someone messed up in regions.py or locations.py

Reply:
Exactly, it only needs to run in debug and can be skipped when frozen to an .exe

Reply:
i see, i do hope it will be run in debug than, maybe the unit tests run in debug? in that case its should be save gaurded